olive/common/ort_inference.py

```diff
@@ -43,8 +43,8 @@ def get_ort_inference_session(
             sess_options.add_session_config_entry(key, value)
 
     if isinstance(execution_provider, list):
-        # execution_provider may be a list of tuples where the first item in each tuple is the EP name
-        execution_provider = [i[0] if isinstance(i, tuple) else i for i in execution_provider]
+        # execution_provider may be a list of tuples/lists where the first item in each tuple is the EP name
+        execution_provider = [i[0] if isinstance(i, tuple) or isinstance(i, list) else i for i in execution_provider]
```

Comment:
isinstance(i, (tuple, list)) should be concise

Reply:
Updated.